src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ CHANGELOG
  * renamed `PostResponseEvent` to `TerminateEvent`
  * added `HttpClientKernel` for handling requests with an `HttpClientInterface` instance
  * added `trace_header` and `trace_level` configuration options to `HttpCache`
+ * Getting the container from a non-booted kernel Sis deprecated.
```

Comment:
typo

Reply:
@Simperfit still there :) ("Sis" -> "is")

Reply:
oops